ui/app/templates/components/transit-key-action/rewrap.hbs

```diff
@@ -37,6 +50,52 @@
         </div>
       </div>
     {{/if}}
+    {{#if (or (eq @key.type "rsa-2048") (eq @key.type "rsa-3072") (eq @key.type "rsa-4096"))}}
+      <div class="field">
+        <label for="decrypt_padding_scheme" class="is-label">Decrypt Padding Scheme</label>
+        <div class="control is-expanded">
+          <div class="select is-fullwidth">
+            <select
+              name="decrypt_padding_scheme"
+              id="decrypt_padding_scheme"
+              data-test-decrypt-padding-scheme="true"
+              onchange={{action (mut @decrypt_padding_scheme) value="target.value"}}
+            >
+              {{#each (array "oaep" "pkcs1v15") as |scheme|}}
+                <option
+                  selected={{if @decrypt_padding_scheme (eq @decrypt_padding_scheme scheme) (eq scheme "oaep")}}
```

Comment:
same comment here about setting the default for `decrypt_padding_scheme` in the `transit-key-actions.js` file and simplifying this conditional to just `{{eq @decrypt_padding_scheme scheme}}`